src/components/file-handler/FileHandler.a11y.spec.jsx

```diff
@@ -0,0 +1,65 @@
+import * as React from 'react';
+import {render} from '@testing-library/react';
+import FileHandler from './FileHandler';
+import {testA11y} from '../../axe';
+import userEvent from '@testing-library/user-event';
+
+describe('FileHandler', () => {
+  it('fires onClick on click, space and enter', () => {
+    const handleOnClick = jest.fn();
+    const fileName = 'file name';
+    const fileHandler = render(
+      <FileHandler onClick={handleOnClick}>{fileName}</FileHandler>
+    );
+
+    userEvent.click(fileHandler.getByText(fileName));
+    fileHandler.getByText(fileName).focus();
+    userEvent.keyboard('{space}');
+    userEvent.keyboard('{enter}');
+
+    expect(handleOnClick).toHaveBeenCalledTimes(3);
+  });
+
+  it('passes src to link href and does not act on space', () => {
```

Comment:
Could we please clarify this test a bit?

I understand it as follows:
- When I pass src to file handler, src is being passed to internal link element
- When I pass src to file handler, hitting spacebar doesn't do anything
- What if I hit enter? Should I do something? maybe we cover both cases in this test?

Reply:
I hope now it's a bit more clear